### ISSUE TITLE
Focus visible

### ISSUE THREAD

**Issue by ashleypeacock** [melissaegg/trauma-informed-design-resources#11]
### Issue
Focus is given to the resource elements by providing a black outline. However, focus is not always visible. Once focus has been given, tabbing hides focus indicator.

Step 1: focus is visible

<img width="745" alt="image" src="https://github.com/user-attachments/assets/1b91cfa9-3e5b-4441-92ac-29f6b020a6ac" />

Step 2 after a second tab: focus is no longer visible
<img width="740" alt="image" src="https://github.com/user-attachments/assets/3c1e698f-bfc1-4902-923d-3af48e1ce374" />

**Expected behaviour**: each keyboard tab should move focus to each element (resource) in the list
